ui/src/tool-panel/location-track/splitting/location-track-split.tsx

```diff
@@ -32,6 +32,8 @@ type SplitProps = EndpointProps & {
     duplicateOf: LocationTrackId | undefined;
     nameErrors: ValidationError<Split>[];
     descriptionErrors: ValidationError<Split>[];
+    nameRef?: React.RefObject<HTMLInputElement>;
+    descriptionBaseRef?: React.RefObject<HTMLInputElement>;
```

Comment:
Pihviä. Focusointi hoidetaan reffien avulla

Reply:
Olikos näiden reffien tarpeellista hyväksyä undefinedit?

Reply:
Ei oikeastaan. Toistaiseksi ei myöskään ole tarvetta käyttää näitä Split-komponentteja missään infoboksin ulkopuolella, joten nämä saavat aina arvot. Nuo optionalityt ovat myös jäännettä devausajalta -> ne poistoon